core/server/helpers/index.js

```diff
@@ -93,13 +94,16 @@ registerHelpers = function (adminHbs) {
     registerThemeHelper('author', coreHelpers.author);
     registerThemeHelper('body_class', coreHelpers.body_class);
     registerThemeHelper('content', coreHelpers.content);
+    registerThemeHelper('img_url', coreHelpers.img_url);
     registerThemeHelper('date', coreHelpers.date);
     registerThemeHelper('encode', coreHelpers.encode);
     registerThemeHelper('excerpt', coreHelpers.excerpt);
     registerThemeHelper('foreach', coreHelpers.foreach);
     registerThemeHelper('has', coreHelpers.has);
     registerThemeHelper('is', coreHelpers.is);
     registerThemeHelper('image', coreHelpers.image);
+    registerThemeHelper('feature_image', coreHelpers.feature_image);
+    registerThemeHelper('profile_image', coreHelpers.image);
```

Comment:
I don't think these 2 lines are needed? Adding the attributes to the toJSON function should be enough 😄